src/ResourceManager/IotCentral/Commands.IotCentral/AzureRM.IotCentral.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.0.0'
+ModuleVersion = '1.0.1'
```

Comment:
Should this be a minor version bump instead? it isn't just a bug fix.

Reply:
@emgarten resolved with ps team